samples/CommunityToolkit.Maui.Sample/ViewModels/Views/AvatarView/AvatarViewBindablePropertiesViewModel.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections.ObjectModel;
+using CommunityToolkit.Maui.Sample.Models;
+
+namespace CommunityToolkit.Maui.Sample.ViewModels.Views.AvatarView;
+
+public class AvatarViewBindablePropertiesViewModel : BaseViewModel
+{
+	public ObservableCollection<AvatarModel> AvatarList { get; } = new()
+	{
+		new AvatarModel
+		{
+			Text = "SW", // Sam Worthington
```

Comment:
Can the comment be in Description property?
Please add Description property in the sample as well

Reply:
The comment, and even the text used was just an amusement in that they are the actors' names from the 2009 movie "Avatar".  They don't add anything and could be any text at all (such as VA, BM, SL, PJ, GV), but I thought it might be regarded as an 'easter egg' for any user looking at the source code.

I have however added a description property to the model and bind them to the _SeamanticProperties_ in the sample view.